### DISCUSSION TITLE
Pinia not working on with Nuxt3

### DISCUSSION THREAD

**Discussion by dbzx10299** [nuxt/nuxt#19307] [Questions]
I just started using nuxt and I reached a point I needed to use a central store so I went over to the pinia docs and did as they said but I keep getting errors in my console.

I followed the nuxt docs to create a project with the nuxi cli, and I have a basic package json file

According to the docs we need to run this command
```bash
npm install pinia @pinia/nuxt
```

There are further instructions that this should be added to the `package.json`
```
"overrides": {
  "vue": "latest"
}
```

After doing this I got an error
```
npm ERR! Invalid comparator: latest
```

If I remove the overrides vue latest from package json I get this error
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: undefined@undefined
npm ERR! Found: vue@3.2.47
npm ERR! node_modules/vue
npm ERR!   peer vue@"^3.2.47" from @nuxt/vite-builder@3.2.2
npm ERR!   node_modules/@nuxt/vite-builder
npm ERR!     @nuxt/vite-builder@"3.2.2" from nuxt@3.2.2
npm ERR!     node_modules/nuxt
npm ERR!       dev nuxt@"^3.2.2" from the root project
npm ERR!   peer vue@">=2.7 || >=3" from @unhead/vue@1.1.11
npm ERR!   node_modules/@unhead/vue
npm ERR!     @unhead/vue@"^1.1.9" from @vueuse/head@1.1.9
npm ERR!     node_modules/@vueuse/head
npm ERR!       @vueuse/head@"^1.0.26" from nuxt@3.2.2
npm ERR!       node_modules/nuxt
npm ERR!         dev nuxt@"^3.2.2" from the root project
npm ERR!   6 more (@vitejs/plugin-vue, @vitejs/plugin-vue-jsx, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! pinia@"*" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: vue@2.6.14
npm ERR! node_modules/vue
npm ERR!   peer vue@">= 2.5 < 2.7" from @vue/composition-api@1.7.1
npm ERR!   node_modules/@vue/composition-api
npm ERR!     peerOptional @vue/composition-api@"^1.4.0" from pinia@2.0.32
npm ERR!     node_modules/pinia
npm ERR!       pinia@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

I did try deleting the `package-lock.json` and the `node_modules` folder but this did not work.

**Comment by AloisSeckar** [comment]
Try changing `latest` to more specific `^3.2.47`.

This helped us on the project on Friday. But I am not sure why exactly this is happening now.

Or you can also try to remove the override and see if `npm install` runs. I have observed, that it only matters in certain combinations of dependencies, usually coming from some 3rd party packages.

**Reply by dbzx10299** [reply]
When the override is removed the the bottom error shows from my above post. 

I got it to work by running 
```
npm i —force pinia @pinia/nuxt
```
I‘ve been having quite a few issues lately following docs and I get errors left and right

**Reply by AloisSeckar** [reply]
Sorry, didnt read it carefully to the end. Glad you managed to solve the problem.
